packages/@office-iss/react-native-win32-tester/src/js/examples-win32/Accessibility/AccessibilityExampleWin32.tsx

```diff
@@ -474,10 +474,28 @@ const AccessibilityInfoExample: React.FunctionComponent<{}> =() => {
   const onClick = React.useCallback(() => {
     AccessibilityInfo.announceForAccessibility('AccessibilityInfo announcement succeeded!');
   }, []);
+
+  const onClickDelayed = React.useCallback(() => {
+    setTimeout(() => {
+      AccessibilityInfo.announceForAccessibilityWithOptions(
+        'AccessibilityInfo announcement succeeded!',
+        // @ts-ignore
```

Comment:
I added this temporarily to get passed this error: `error TS2345: Argument of type '{ nativeID: string; }' is not assignable to parameter of type '{ queue?: boolean; }'.
  Object literal may only specify known properties, and 'nativeID' does not exist in type '{ queue?: boolean; }'.`
  
I'm not quite sure how to get it to resolve to the right types.

Reply:
You'll need to modify/override the AccessibilityInfo.d.ts from core to ensure that the type information is exposed for typescript users.